spec/models/framework/management_charge_calculator/flat_rate_spec.rb

```diff
@@ -12,11 +12,31 @@
     end
 
     context 'with the calculation field overriden' do
-      let(:calculator_arguments) { { percentage: BigDecimal('1.5'), column: 'Some field' } }
+      let(:calculator_arguments) { { percentage: BigDecimal('0.5'), column: 'Some field' } }
       let(:entry) { FactoryBot.build(:submission_entry, total_value: 1234, data: { 'Some field' => 180.83 }) }
```

Comment:
Each context now has its own `entry`, so this `let` is redundant

Reply:
👍🏽